lms/djangoapps/verify_student/services.py

```diff
@@ -13,10 +13,40 @@
 from student.models import User, CourseEnrollment
 from lms.djangoapps.verify_student.models import VerificationCheckpoint, VerificationStatus, SkippedReverification
 
+from .models import SoftwareSecurePhotoVerification
```

Comment:
Please add a TODO comment to clarify that this is not what we would ideally be doing.